PS8lite.Rmd

```diff
@@ -201,9 +202,22 @@ Fit one Random Forest model `model_rf` using any approach you want.
 Using `model_rf` make predictions and submit to Kaggle.
 
 ```{r}
+#Make predictions by evaluating randomForests model on test dataset
+test <- test %>% 
+  mutate(
+    logSalePrice_hat_rf = predict(object = model_rf, newdata = test),
+    SalePrice_hat_rf = exp(logSalePrice_hat_rf) -1)
+#glimpse(test)
+
+#Kaggle submission code
+submission_rf <- test %>%
+  mutate(SalePrice = SalePrice_hat_rf) %>%
+  select(Id, SalePrice)
 
+#Write csv file to sumbit to Kaggle and get RMSLE score for our model
+write_csv(submission_rf, "data/submission_rf.csv")
```

Comment:
Hi, everything looks good. Here is an example comment that I would leave as part of a “code review” if this were real life. Then you would make the requested changes to this PR and then I would merge your changes into my master copy of the repo. I won't be doing that last step in this case.